doc/modules/ROOT/pages/index.adoc

```diff
@@ -107,15 +107,17 @@ $ java -Dserver.port=8080 -jar target/springboot-tomcat-session-replication-0.1.
 Open another terminal session and start the second application:
 [source,bash]
 ----
-java -Dserver.port=8081 -jar target/springboot-tomcat-session-replication-0.1.0.jar
+$ java -Dserver.port=8081 -jar target/springboot-tomcat-session-replication-0.1.0.jar
```

Comment:
nit: I'd avoid adding `$` if it's a one-liner command. Because it's more difficult to copy and execute in your environment.

Reply:
You are right, removed all.